sgt-back-end/index.js

```diff
@@ -152,7 +152,7 @@ app.delete('/api/grades/:gradeId', (req, res, next) => {
   const params = [req.params.gradeId];
   db.query(sql, params)
     .then(result => {
-      return res.sendStatus(204);
+      return res.sendStatus(404);
```

Comment:
I mean that `result.rows` will be empty if nothing was deleted, but it will have one object in it otherwise. You want to tell the client whether or not the target record was successfully deleted. You need a conditional.